src/pdb.rs

```diff
@@ -56,7 +64,10 @@ impl<'s, S: Source<'s> + 's> PDB<'s, S> {
     /// * `Error::UnrecognizedFileFormat` if the `Source` does not appear to be a PDB file
     /// * `Error::IoError` if returned by the `Source`
     /// * `Error::PageReferenceOutOfRange`, `Error::InvalidPageSize` if the PDB file seems corrupt
-    pub fn open(source: S) -> Result<PDB<'s, S>> {
+    pub fn open(source: S) -> Result<PDB<'s, S>>
+    where
+        S: Send,
```

Comment:
I'm worried that not all consumers can provide a `Send` parameter here.

It would be nice if we could find a way to make PDB send whenever S is Send.

Reply:
I think I've seen ways of doing this with trait objects, though (if I am remembering correctly) they are understandably hacky/cumbersome. It would be easier if this didn't store a `dyn Msf`. There's only one implementation of that, so we could avoid the trait object fairly easily...

Reply:
TBH as of these changes, `Source` must be able to provides `Send + Sync` `SourceViews`, so I'd be surprised if `Source` was not at least `Send` too.